packages/vinxi/lib/file-system-router.js

```diff
@@ -76,21 +77,18 @@ export class BaseFileSystemRouter extends EventTarget {
 	}
 
 	glob() {
-		if (os.platform() === "win32") {
-			return fg.convertPathToPattern(this.config.dir + "//**//*");
-		} else {
-			return (
-				join(this.config.dir, "**/*") + `.{${this.config.extensions.join(",")}}`
-			);
-		}
+		return posix.join(
+			fg.convertPathToPattern(normalize(this.config.dir)), 
+			"**/*"
+		) + `.{${this.config.extensions.join(",")}}`;
 	}
 
 	/**
 	 * @returns {Promise<any[]>}
 	 */
 	async buildRoutes() {
 		await init;
-		glob(this.glob()).forEach((src) => {
+		glob(this.glob().replace(/\\/g, "\\\\")).forEach((src) => {
```

Comment:
For whatever reason the escaped pattern returned by `fg.convertPathToPattern` does not work with `fast-glob` but works with `micromatch`. The escape character `\\` requires to be double escaped for `fast-glob`.

Reply:
The goal of this PR is to get vinxi to run on Windows. Not fixing edge cases of fast-glob here. I will open a separate issue for this.